### PR TITLE
Initial setup pages

### PR DESCRIPTION
**Pull request by amirine**

### CODE REVIEW

File: game_project/settings.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'game.apps.GameConfig' # или лучше просто 'game'?
```
Comment (Satevg) [inline game_project/settings.py]: usually just game :+1:

**Review by Satevg — COMMENTED** [review]
Looks good, but there should be a `requirements.txt` file, that will have all requirements

The first example I have found in internet is this one: https://github.com/gothinkster/django-realworld-example-app/blob/master/requirements.txt

So in your requirements.txt there will be only 1 line - Django==3.x.x - I don't know exact version.

To find out what version you have - run `pip freeze` (under virtualenv)

File: game/templates/game/main_page.html [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+        @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@800&family=Roboto:wght@500&display=swap');
+    </style>
+</head>
+<body>
+
+
+<!-- Navbar -->
+<nav class="navbar navbar-expand bg-danger">
+        <a class="navbar-brand text-white" href="#">
+            <img src="https://www.pngkey.com/png/full/272-2725717_computer-vision-computer-vision-icon-png-white.png"
```
Comment (Satevg) [inline game/templates/game/main_page.html]: Could you please use template inheritance here?
https://docs.djangoproject.com/en/3.2/ref/templates/language/#template-inheritance

I see that you have similar pieces in both templates. After splitting, there should be 3 templates - base.html, main_page.html and game_detail_page.html (and no html code repeated)

**Comment by Satevg** [comment]
Not for this PR, but maybe for the following, please read about `.gitignore` file and include it into the next PR :raised_hands:

File: game/templates/game/main_page.html (hunk 2 above)
Comment (amirine) [reply game/templates/game/main_page.html]: Got it, I'll fix it

File: game_project/settings.py (hunk 1 above)
Comment (amirine) [reply game_project/settings.py]: Okay

**Comment by amirine** [comment]
> Not for this PR, but maybe for the following, please read about `.gitignore` file and include it into the next PR 🙌

It was added with the initial commit, didn't include it into this pull request

**Review by Satevg — APPROVED** [review]
Nice. Feel free to merge this in.

**Pull request merged**